jenkins/databricks/params.py

```diff
@@ -60,6 +61,8 @@
         build_profiles = arg
     elif opt in ('-j', '--jarpath'):
         jar_path = arg
+    elif opt in ('-f', '--sparkconf'):
```

Comment:
if this can take multiple confs we should clarify this more. Add a comment and make sure format is clear.
Other options here might be to comma separate the confs just being spark.foo=1,spark.bar=2.  that might also simplify your parsing for PYSP_TEST_ and not have to have all the extra characters passed along for --conf


Another option might be to use optparse which I believe would allow you to specify the same parameter multiple times -f spark.foo=1 -f spark.bar=2. that might be bigger change though.

Reply:
Yes, I'd like to use comma separate list of spark configurations `spark.foo=1,spark.bar=2` 
Updated